### DISCUSSION TITLE
AI tools implementation

### DISCUSSION THREAD

**Discussion by HichOoMa** [alagrede/znote-app#86] [Ideas]
I think it will be a great feature to add the ability to use AI tools such as Grammarly or killbot that will help you write better sentence and correct grammar mistakes

**Comment by alagrede** [comment]
Hi @HichOoMa,
Thank you for your suggestion! I will investigate the tools you mentioned. I'm currently working on a new WYSIWYG editor for the next release.  I think these AI use cases will be perfectly suited to this new editor 😊

**Comment by alagrede** [comment]
It is now possible to rewrite selected texts with AI tools. Feel free to reopen if you need more help on this topic. 😊
